Cargo.toml

```diff
@@ -32,6 +32,7 @@ authors = ["NVIDIA Inc. <sw-dl-dynamo@nvidia.com>"]
 license = "Apache-2.0"
 homepage = "https://github.com/ai-dynamo/dynamo"
 repository = "https://github.com/ai-dynamo/dynamo.git"
+readme = "README.md"
```

Comment:
> If no value is specified for this field, and a file named README.md, README.txt or README exists in the package root, then the name of that file will be used.

Not harmful to add, but this should already work without being specified according to the docs: https://doc.rust-lang.org/cargo/reference/manifest.html#the-readme-field. Was that not the case?

Reply:
Hm let me try taking it out; I think I originally ran into an issue, but let me see if I can reproduce it

Reply:
No big deal - just curious when I saw it.